api/v1alpha1/bgppeer_webhook.go

```diff
@@ -89,6 +92,25 @@ func (bgpPeer *BGPPeer) validateBGPPeer(existingBGPPeersList *BGPPeerList) error
 	return err
 }
 
+func (bgpPeer *BGPPeer) validateBGPPeersKeepaliveTime(existingBGPPeersList *BGPPeerList) *field.Error {
+	holdTime := bgpPeer.Spec.HoldTime
+	keepaliveTime := bgpPeer.Spec.KeepaliveTime
+
+	// Keepalivetime is not set then return w/o doing any validation
```

Comment:
This makes a lot of sense but may be surprising to new readers. 
Mind extending the comment with the reason behind it?